bundles/org.openhab.core.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttBrokerConnection.java

```diff
@@ -438,6 +445,7 @@ public void setLastWill(@Nullable MqttWillAndTestament lastWill, boolean applyIm
      *
      * @param lastWill The last will object or null.
      */
+    @Deprecated
```

Comment:
Is is the LW handled now instead? Could you maybe if time permits add a documentation line

Reply:
Added errorneously.